appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -50,20 +53,24 @@ function Get-CaseSensitivePath($pathName){
 $sarifLog = (Get-Content $logFilePath) -join "`n" | ConvertFrom-Json
 
 for ($i = 0; $i -lt $sarifLog.runs.Count; $i++){
-  $sarifLog.runs[$i].files.PSObject.Properties | foreach-object{
-    $key = $_.Name
-    $content = $_.Value
-    if($content.uri){
-        $content.uri = Rebase-Uri $content.uri
-    }
-
-    if($content.parentKey){
-      $content.parentKey = Rebase-Uri $content.parentKey
+  if($sarifLog.runs[$i].files){
+    $sarifLog.runs[$i].files.PSObject.Properties | foreach-object{
+      $key = $_.Name
+      $value = $_.Value
+      if($key -and $content){
```

Comment:
You changed the variable you assigned to, but not any of the _uses_ of that variable. This presumably doesn't run correctly.